aten/src/ATen/FunctionalInverses.cpp

```diff
@@ -78,6 +78,11 @@ Tensor FunctionalInverses::_fw_primal_inverse(const at::Tensor& base, const at::
     return Tensor();
 }
 
+Tensor FunctionalInverses::_make_dual_inverse(const at::Tensor& base, const at::Tensor& mutated_view, const at::Tensor& tangent, int64_t level) {
```

Comment:
Where is that used? Or is there some automatic registration system here?

Reply:
It is referenced in an auto generated function for the functionalization pass, so I had to add it to get it to compile. Maybe @bdhirsh knows more?